src/util/airtable.ts

```diff
@@ -0,0 +1,26 @@
+import Airtable from 'airtable';
+import { AIRTABLE_API_KEY, AIRTABLE_BASE_ID } from './secrets';
+
+interface PipelineSignUpParams {
+    Email: string;
+    Date: string | Date;
+    AcceptUpdates?: boolean;
+    [key: string]: any;
+}
+
+const formatDate = (datestring: string | Date) => {
+    const date = new Date(datestring);
+    return date.getMonth() + '/' + (date.getDay() + 1) + '/' + date.getFullYear();
```

Comment:
@GarfDev I've found this date formatting to be incorrect in my earlier review. Sorry I did not notify you about that!

Please correct me if I'm wrong, but:
1. I believe you should use `getDate()` to get the day of the month as an int. 
2. `getMonth` should be incremented with 1 since it will return the month as an int, but starting at 0 (and there is not month 0 in numeric date formatting).

Reply:
Okay, just google that [issue](https://stackoverflow.com/questions/15799514/why-does-javascript-getmonth-count-from-0-and-getdate-count-from-1). Thank you Peter